user/usecase/changePassword/changePassword.go

```diff
@@ -0,0 +1,44 @@
+package changePassword
+
+import (
+	"github.com/WeisswurstSystems/WWM-BB/user"
+	"github.com/WeisswurstSystems/WWM-BB/user/usecase"
+	"github.com/WeisswurstSystems/WWM-BB/user/usecase/authenticate"
+)
+
+// ChangePasswordUseCase changes the password of the user.
+//
+// The password must be repeated to ensure no type error.
+type ChangePasswordUseCase interface {
+	ChangePassword(Request) error
+}
+
+// Request for changing a password.
+type Request struct {
+	Login            user.Login `json:"login"`
+	Password         string     `json:"password"`
+	PasswordRepeated string     `json:"passwordRepeated"`
```

Comment:
Hier verlangst du es, nutzt es aber nicht im Code. Und es ist die Frage, ob das jetzt mehr Frontend oder Backend-Logik ist?

Reply:
hab ich bestimmt vergessen, wollte is im backend haben